index.js

```diff
@@ -76,7 +76,8 @@ class CommandPrompt extends InputPrompt {
           console.log()
           process.stdout.cursorTo(0)
           console.log(chalk.red('>> ') + chalk.grey('Available commands:'))
-          console.log(CommandPrompt.formatList(ac.matches))
+          console.log(CommandPrompt.formatList(
+              ac.matches.map(function (value) { return value.split(" ").slice(-1).join(); })))
```

Comment:
I am using this plugin in an app where I prefer to see the entire command. What if we add an option so that, instead of doing just the mapping on the matches we have:
```
            this.opt.lastOnly
                  ? ac.matches.map(value => {
                    return value.split(" ").slice(-1).join()
                  })
                  : ac.matches
```

Reply:
Although I personally don't see the value in repeating already written commands (and possibly options) as seen in my first screenshot, having an option to change the beheavior is a decent compromise. I'd say go for the configurable option.

However I am curious, how do you consider exposing that particular option? Since you enable the plugin using the following line
```
inquirer.registerPrompt('command', require('inquirer-command-prompt'))
```
there is no actual object to work with.

Reply:
Never mind, I have verified that that command breaks the general behavior in many cases. I will think about the problem and find a solution for that with a better command. For now I will close this PR.